model/map/src/main/java/org/keycloak/models/map/authorization/MapPolicyStore.java

```diff
@@ -155,9 +152,9 @@ public Policy findByName(String name, String resourceServerId) {
         }
 
         return paginatedStream(tx.getUpdatedNotRemoved(mcb)
-                .sorted(AbstractPolicyEntity.COMPARE_BY_NAME), firstResult, maxResult)
-                .map(MapPolicyEntity::getId)
-                .map(UUID::toString)
+                .sorted(MapPolicyEntity.COMPARE_BY_NAME), firstResult, maxResult)
+                .map(MapPolicyEntity<K>::getId)
+                .map(K::toString)
```

Comment:
Should we use this?:
```suggestion
                .map(policyStore.getKeyConvertor()::keyToString)
```

Reply:
good catch, this will be fixed later